src/store/cart/cart.ts

```
@@ -0,0 +1,120 @@
+            .map(item => {
+              if (item.id === itemId) {
+                const newQuantity = (item.cartCount || 0) - 1
+
+                return newQuantity > 0
+                  ? { ...item, cartCount: newQuantity }
+                  : { ...item, cartCount: 0 }
+              }
+
+              return item
+            })
+            .filter(item => item.cartCount || 0 > 0)
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)